commit bd5326bdf072a38a96da626d4976c8ad9b7d491c (refs/changes/91/2291/3)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-07-31T17:47:51+00:00 (2 years, 2 months ago)
    
    TOOLS-1824 sdcadm update-other should not do anything for VOLAPI
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
    Approved by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/tools/setup/setup.sh b/tools/setup/setup.sh
index 8262643..f5fc4bd 100644
--- a/tools/setup/setup.sh
+++ b/tools/setup/setup.sh
@@ -288,13 +288,6 @@ upgrade_core_service_to_latest_branch_image "cloudapi" "tritonnfs"
 # volapi" but the VOLAPI zone may need to be updated to the latest version.
 upgrade_core_service_to_latest_branch_image "volapi" "tritonnfs"
 
-echo "Running other experimental migration processes"
-# "sdcadm experimental update-other" takes care of doing things such as adding a
-# VOLAPI_SERVICE key in the sdc's SAPI application metadata, running VMAPI
-# migrations, etc., which are not done by "sdcadm experimental volapi", so this
-# needs to be run every time to make sure everything is up to date.
-sdcadm experimental update-other
-
 # Needed to add additional programs in the HN's GZ, such as sdc-volapi
 upgrade_gz_tools_to_latest_branch_image tritonnfs
 
